spec/lucky/action_rendering_spec.cr

```diff
@@ -206,4 +212,10 @@ describe Lucky::Action do
     response.status.should eq 200
     response.content_type.should eq "text/html"
   end
+
+  it "renders from a fallback" do
+    response = Rendering::FallbackRoute.new(build_context, params).call
+    response.body.should eq "Hey, you found me!"
+    response.status.should eq 200
+  end
```

Comment:
What do you think about moving to a `fallback_action_spec.cr`? And then adding another spec for `it "does not generate route helpers"` and it checks that the fallback action does not `respond_to?` `route`, `with`, `url`, `path` or `url_without_query_params`